requirements.txt

```diff
@@ -1,5 +1,5 @@
 altair>=4.2.1,<5.0.0
-Click>=7.1.2,!=8.1.4 # 8.1.4 has a mypy typing bug (https://github.com/pallets/click/issues/2558)
+Click>=7.1.2
```

Comment:
Type checking fails with Click 8.1.4 and our static analysis checks fail. CI would die if Click==8.1.4 is installed.

Reply:
It looks like Click 8.1.5 only has type changes (including addition of tests for this):
https://click.palletsprojects.com/en/8.1.x/changes/

There shouldn't be any runtime issues so we can remove this but I'd add a comment saying it dies but only at type checking time.